pkg/logs/input/tailer/tailer_nix.go

```diff
@@ -0,0 +1,101 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build !windows
```

Comment:
agent doesn't compile on unix:
```
# github.com/DataDog/datadog-agent/pkg/logs/input/tailer
pkg/logs/input/tailer/tailer_nix.go:11:2: imported and not used: "fmt"
pkg/logs/input/tailer/tailer_nix.go:15:2: imported and not used: "sync"
pkg/logs/input/tailer/tailer_nix.go:16:2: imported and not used: "sync/atomic"
pkg/logs/input/tailer/tailer_nix.go:17:2: imported and not used: "time"
pkg/logs/input/tailer/tailer_nix.go:21:2: imported and not used: "github.com/DataDog/datadog-agent/pkg/logs/config"
pkg/logs/input/tailer/tailer_nix.go:23:2: imported and not used: "github.com/DataDog/datadog-agent/pkg/logs/message"
pkg/logs/input/tailer/tailer_nix.go:93:5: undefined: inode
```

Happy to fix it myself if you don't have a working unix environment